src/System.Windows.Forms.Primitives/src/Interop/User32/Interop.MDITILE.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum MDITILE : uint
```

Comment:
```suggestion
		[Flags]
        public enum MDITILE : uint
```
![image](https://user-images.githubusercontent.com/4403806/74203130-a98a4900-4cc2-11ea-9eaa-ee1b99a86b52.png)